backend/src/openbeheer/clients.py

```diff
@@ -0,0 +1,60 @@
+from functools import cache
+from django.core.exceptions import ImproperlyConfigured
+from django.db.models.signals import post_save, post_delete
+from django.dispatch import receiver
+from django.utils.translation import gettext_lazy as __
+
+from ape_pie import APIClient
+from zgw_consumers.client import build_client
+from zgw_consumers.constants import APITypes
+from zgw_consumers.models import Service
+
+
+@cache
+def zrc_client() -> APIClient:
+    try:
+        client = build_client(
+            Service.objects.get(api_type=APITypes.zrc),
+        )
+    except Service.DoesNotExist:
+        raise ImproperlyConfigured(__("No ZRC service configured")) from None
```

Comment:
I am not sure how I feel about `__` :smile:

Reply:
Yeah. I think it's unfortunate that Django uses `_` as the name for gettext; `_` has always meant unused, or "result of previous expression in the REPL".
Maybe I'm just too scared of shadowing the gettext value for `_`. I'll change it back.